lib/root.html.erb

```diff
@@ -1,7 +1,18 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="UTF-8"><meta http-equiv="refresh" content="5;url=/current/">
+    <title>Welcome to Go Continuous Delivery API Documentation</title>
```

Comment:
"GoCD's API documentation" should be fine. We don't need to welcome anyone in a title, I feel.